lib/charms/mysql/v0/mysql.py

```diff
@@ -90,7 +90,7 @@ def wait_until_mysql_connection(self) -> None:
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 26
+LIBPATCH = 28
```

Comment:
i believe this should be libpatch 27 (on the vm charm - which where the charm lib gets published)